spec/system/home_spec.rb

```diff
@@ -174,4 +174,26 @@ def ie_alert_box_xpath
 
     expect(page).not_to have_css(".title", text: "Featured")
   end
+
+  describe "Header Card" do
+    scenario "if there is header card with link, the link content is rendered" do
+      create(:widget_card,
+        title: "Card text",
+        description: "Card description",
+        link_text: "Link text",
+        link_url: "consul.dev",
+        header: true
+      )
+
+      visit root_path
+
+      expect(page).to have_css(".link-content", text: "Link text")
+    end
+
+    scenario "if there is header card with link, the link content is not rendered" do
```

Comment:
The name of the scenario is the same as the previous one (with the exception of the ending "not") :thinking:. Also, for this test I take we want to create a card without a link text?